.travis.yml

```diff
@@ -58,7 +58,7 @@ matrix:
 before_script:
 # Init PHP
   - pecl channel-update pecl.php.net
-  - printf "\n" | pecl install imagick
+  - NEXT_WAIT_TIME=0 && until (printf "\n" | pecl install imagick) || [[ $NEXT_WAIT_TIME == "4" ]]; do sleep $(( NEXT_WAIT_TIME++ )); done
```

Comment:
use `travis_retry`? or even just add `|| true` like we do for other non-essential build steps?

Reply:
`travis_retry` sounds like a better idea for this :)